config/environments/development.rb

```diff
@@ -53,4 +53,6 @@
   # Config::Options objects, but we need them as hashes.
   cache_settings = Settings.cache.to_hash
   config.cache_store = cache_settings[:store].to_sym, *cache_settings[:opts]
+
+  GA.tracker = Settings.googleanalytics.tracker
```

Comment:
Turn out `GA` is an alias for `GoogleAnalytics`.  I'll use `GoogleAnalytics` instead of `GA` throughout for clarity, and make a note in the config files that it's defined in the `google-analytics-rails` gem.